model/infinispan/src/main/java/org/keycloak/cluster/infinispan/InfinispanClusterProviderFactory.java

```diff
@@ -195,6 +198,17 @@ public boolean isSupported(Config.Scope config) {
     @Listener
     public class ViewChangeListener {
 
+        @Merged
+        public void mergeEvent(MergeEvent event) {
```

Comment:
Please add more context here why we do this. When looking at the code in this method, people would see what we are doing, but not why. Thanks!